### PR TITLE
Deactivate project and add note to README

### PR DESCRIPTION
**Pull request by davehunt**
Pinging @bobsilverberg and @m8ttyB for review. This pull replaces #108.

### CODE REVIEW

**Comment by m8ttyB** [comment]
r+

/cc @groovecoder

File: README.md [hunk 1]
```diff
@@ -1,103 +1,5 @@
...
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-[MPL]: http://www.mozilla.org/MPL/2.0/
+This repository is no longer being actively maintained, and the contents has
+been removed. Please use the repository history to access the previous content.
+We are not currently executing or developing tests for MDN in this repository.
+You can read more about MDN development and testing in the
+[project documentation](https://kuma.readthedocs.org/en/latest/installation.html).
```
Comment (groovecoder) [inline README.md]: Better url is http://kuma.readthedocs.org/en/latest/tests-ui.html

File: README.md [hunk 2]
```diff
@@ -1,103 +1,5 @@
...
--------
-This software is licensed under the [MPL] 2.0:
-
-    This Source Code Form is subject to the terms of the Mozilla Public
-    License, v. 2.0. If a copy of the MPL was not distributed with this
-    file, You can obtain one at http://mozilla.org/MPL/2.0/.
-
-[MPL]: http://www.mozilla.org/MPL/2.0/
+This repository is no longer being actively maintained, and the contents has
+been removed. Please use the repository history to access the previous content.
```
Comment (stephendonner) [inline README.md]: nit: s/contents has/content has/ (singular)

File: README.md (hunk 1 above)
Comment (davehunt) [inline README.md]: I considered that, but instead decided to link to the top level. There are a couple of test related pages in the documentation and didn't want to deep link to just one (it also makes the link more fragile). That said, it appears I directly linked to the installation page, so I'll change it to simply https://kuma.readthedocs.org

File: README.md (hunk 2 above)
Comment (davehunt) [inline README.md]: +1

**Comment by davehunt** [comment]
Updated, please can I get a final review? @groovecoder I notice that your documentation links to this repository - you may want to remove that. See https://github.com/mozilla/kuma/blob/master/README.rst#getting-started

**Comment by groovecoder** [comment]
r+ and I submitted https://github.com/mozilla/kuma/pull/3356

**Pull request merged**